src/coreComponents/physicsSolvers/NonlinearSolverParameters.hpp

```diff
@@ -337,7 +343,8 @@ ENUM_STRINGS( NonlinearSolverParameters::CouplingType,
 
 ENUM_STRINGS( NonlinearSolverParameters::SequentialConvergenceCriterion,
               "ResidualNorm",
-              "NumberOfNonlinearIterations" );
+              "NumberOfNonlinearIterations",
+              "SolutionIncrements" );
```

Comment:
maybe just `SolutionIncrement` ?

Reply:
prefer `SolutionIncrements` since there are many of them typically